.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+- id: pyright
+  name: pyright
+  entry: pyright .
+  language: node
+  types: [python]
+  additional_dependencies: ['pyright@1.1.97']

```

Comment:
This will require updating every time I release a new version of pyright. I'm not crazy about adding that additional maintenance work. What is the benefit of adding a ".pre-commit-hooks.yaml" to this repo versus documenting the contents of a ".pre-commit-hook.yaml" that someone else would need to check in to their repo to use pyright in their CI? That way, they can decide which version of pyright to use and when to upgrade it.